tasks/Finals.tex

```diff
@@ -21,9 +21,21 @@ \subsection{Task}
 
 \begin{enumerate}[nosep]
 \item \textbf{Closing Furniture:} Doors of the Cabinet as well as the Dishwasher need to be closed.
-\item \textbf{Welcome Guest:} There is an additional Person waiting behind the Exit door.
+\item \textbf{Welcome Guest:} There is an additional Person waiting behind the Exit door. The Person will state their Request only after the door is opened by the robot. As the Position is known, there will be no points awarded for finding this person.
 \end{enumerate}
 
+\subsection{The Show Must Go On}
+To ensure a good experience for the audience the teams are allowed to restart the robot inside the arena.
+\begin{enumerate}[nosep]
+  \item All previously collected points will be kept.
+  \item The tasks in the arena may be moved by the Referee during the restart.
```

Comment:
This sounds weird. Do you mean the order of the tasks can be determined by the referee?

Reply:
Yes your suggestion sound better